static/elements/chromedash-featurelist.js

```diff
@@ -333,11 +330,9 @@ class ChromedashFeaturelist extends LitElement {
 
     this._setOpenFeatures(targetId, true);
 
-    for (let i = 0, f; f = this.filtered[i]; ++i) {
-      if (f.id === targetId) {
-        this._virtualizerEl.scrollToIndex(i);
-        return;
-      }
+    const targetEl = this.shadowRoot.querySelector('#id-' + targetId);
+    if (targetEl) {
+      targetEl.scrollIntoView();
```

Comment:
This would only scroll the item at the top of the screen, which is covered by the header.

I think we can add these below this line to scroll pass the header.
```
const heightOfHeader = $('.main-toolbar').getBoundingClientRect().height;
window.scrollBy(0, heightOfHeader * -1);
```

Reply:
Thank you! That's exactly the kind of help that I needed.